src/Lime.Cli/Options.cs

```diff
@@ -35,5 +42,11 @@ public class Options : IOptions
 
         [Option(HelpText = "The action to be executed in the non-interactive mode.")]
         public string Action { get; set; }
+
+        [Option(HelpText = "The thumbprint of the X509 certificate to be used.")]
+        public string Thumbprint { get; set; }
```

Comment:
Maybe same comment as Minelli´s? Calling it Certificate Thumbprint might help in code readability